runtime/client/src/lib.rs

```diff
@@ -0,0 +1,251 @@
+use async_std::{sync::Mutex, task};
```

Comment:
This file is only used for the tests, right? I would move it to the `integration` folder. Or at least compile it conditionally